### PR TITLE
Fixing peer stop command - issue #886

### PR DESCRIPTION
**Pull request by gennadylaventman**
On running peer side:
1. Peer store its pid into peer.pid file in "peer.fileSystemPath"
2. Once ServerAdmin (admin.go) get StopServer, it removes pid file and executes os.Exit(0)
3. In case of GetStatus it return STARTED, based of that while peer process is still running and able to response to grpc - it seems OK

On another side:
1. Process try to connect to peer using grpc, as it was before. But one it expect to disconnect, as other side is exited.
2. If grpc doesn't work, it look for pid file and try to send SIGTERM to process.

### CODE REVIEW

**Comment by gennadylaventman** [comment]
Signed-off-by: Gennady Laventman gennady@il.ibm.com

File: core/admin.go [hunk 1]
```diff
@@ -59,15 +59,15 @@ func worker(id int, die chan struct{}) {
...
-	die := make(chan struct{})
-	log.Debug("Creating %d workers", viper.GetInt("peer.workers"))
-	for i := 0; i < viper.GetInt("peer.workers"); i++ {
-		go worker(i, die)
-	}
-	runtime.Gosched()
-	<-time.After(1 * time.Millisecond)
-	close(die)
+	status := &pb.ServerStatus{Status: pb.ServerStatus_STARTED}
+	//die := make(chan struct{})
```
Comment (srderson) [inline core/admin.go]: Can this block of commented out code be removed?

**Comment by gennadylaventman** [comment]
@srderson Yes, sure. I will remove it now and commit.

**Comment by gennadylaventman** [comment]
@srderson Done.

**Comment by srderson** [comment]
LGTM. Thanks

**Pull request merged**